Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM python:3.6-alpine
 
 COPY requirements.txt app/
 
+RUN apk --no-cache add gcc musl-dev
```

Comment:
@xmalmed in fact I'm not sure if it should or shouldn't be here, can you please check that? I think that it was your contribution, and maybe I forgot to check and rebase properly.